applications/visualizer/frontend/src/components/ViewerContainer/Neurons.tsx

```diff
@@ -15,6 +15,12 @@ const mapToListItem = (neuron: string, visibility: ViewerData) => ({
   label: neuron,
   checked: Object.values(visibility).every((e) => e === undefined || e.visibility === Visibility.Visible),
 });
+const mapNeuronsToListItem = (neuron: string, visibility: ViewerData) => ({
+  id: neuron,
+  label: neuron,
+  checked: Object.values(visibility).every((e) => e === undefined || e.visibility === Visibility.Visible),
+  color: visibility[ViewerType.ThreeD]?.color || visibility[ViewerType.EM]?.color || "#000000",
```

Comment:
note: I think this is fine for now because we don't (yet) allow different colors in different viewers, but when/if we do, we need to get a consensus on what color should be used for the broader Neuron representation (the color that shows in the sidebar)

Reply:
@afonsobspinto yes, I thought about that also, but I think it's fine at the moment, we will probably need to reflect a little bit more if we can have different color by viewer as it will also impact the synchronizer probably, or the way we retrieve the information.